restToDo/urls.py

```diff
@@ -0,0 +1,24 @@
+"""
+URL configuration for restToDo project.
```

Comment:
can we remove unnecessary comments from all the code

only include the comments that are necessary